src/App.tsx

```diff
@@ -1,58 +1,114 @@
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import 'bulma/bulma.sass';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-import classNames from 'classnames';
+import cn from 'classnames';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { ContextUsers } from './components/UsersContext';
+import { User } from './types/User';
+import { getUsers } from './components/api/getUsers';
+import { getPosts } from './components/api/getPosts';
+import { Post } from './types/Post';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [isLoadingPosts, setIsloadingPosts] = useState(false);
+  const {
+    posts,
+    setPosts,
+    openSidebar,
+    visiblePost,
+    setVisiblePost,
+    showErrOnLoad,
+    setShowErrOnLoad,
+    userSelected,
+    selectedPost,
+  } = useContext(ContextUsers);
+
+  //@dev for load users on mount
```

Comment:
delete comments

Reply:
These comments are necessary for me, and  other mentor sad to me that's 100% ok , so who are right?